pymc3/tests/test_step.py

```diff
@@ -342,9 +341,9 @@ def test_binomial(self):
 class TestNutsCheckTrace(object):
     def test_multiple_samplers(self):
         with Model():
-            prob = Beta('prob', alpha=5, beta=3)
+            prob = Beta('prob', alpha=5., beta=3.)
             Binomial('outcome', n=1, p=prob)
-            with warnings.catch_warnings(record=True) as warns:
+            with pytest.warns(None) as warns:
```

Comment:
ok, that wasn't it. This failure is a bit of a mystery to me, but it's not related to the PR anyway.